server/service/AuthorizationApplication.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @file This module provides functionality to monitor the whether the registered application are approved within the required time.
+ * @module MonitorTypeApprovalChannel
+ **/
+
+
+const fs = require('fs');
+const { getFilePath } = require('onf-core-model-ap/applicationPattern/onfModel/models/profile/FileProfile');
+
+
+const profile = require('onf-core-model-ap/applicationPattern/onfModel/models/profile/FileProfile');
+const { resolve } = require('path');
+
+const administratorList = 'administrator-credential-list';
+const authorizationValue ='authorization';
+const allowedMethodsValue ='allowed-methods';
+
+
+
```

Comment:
Please add short comments for all the methods to understand about the method . Kindly change the file to path ../individualService folder